internal/rootcoord/root_coord.go

```diff
@@ -101,15 +101,10 @@ type Core struct {
 	//setMsgStreams, send drop partition into dd channel
 	SendDdDropPartitionReq func(ctx context.Context, req *internalpb.DropPartitionRequest, channelNames []string) error
 
-	// if rootcoord create segment, datacoord will put segment msg into this channel
-	DataCoordSegmentChan <-chan *ms.MsgPack
-
-	// if segment flush completed, data node would put segment msg into this channel
-	DataNodeFlushedSegmentChan <-chan *ms.MsgPack
-
 	//get binlog file path from data service,
 	CallGetBinlogFilePathsService func(ctx context.Context, segID typeutil.UniqueID, fieldID typeutil.UniqueID) ([]string, error)
 	CallGetNumRowsService         func(ctx context.Context, segID typeutil.UniqueID, isFromFlushedChan bool) (int64, error)
+	CallGetFlushedSegments        func(ctx context.Context, collID, partID typeutil.UniqueID) ([]typeutil.UniqueID, error)
```

Comment:
Please rename this func to "CallGetFlushedSegmentsService", to align to the naming style "CallGetXXXService"